src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Operations/Export/ExportAnonymizerTests.cs

```diff
@@ -0,0 +1,194 @@
+            IAnonymizer anonymizer = await CreateAnonymizerFromConfigContent(configurationContent);
+
+            ResourceElement resourceElement = anonymizer.Anonymize(new ResourceElement(patient.ToTypedElement()));
+            Patient anonymizedResource = resourceElement.Instance.ToPoco<Patient>();
+            Assert.Equal("3cafe40f92be6ac77d2792b4b267c2da11e3f3087b93bb19c6c5133786984b44", anonymizedResource.Id);
+        }
+
+        [Fact]
+        public async Task GivenDateShiftAnonymizationConfig_WhenAnonymizeResource_ThenShiftedNodeShouldBeReturned()
+        {
+            string configurationContent =
+            @"
+            {
+              ""fhirPathRules"": [
+                {
+                  ""path"": ""Patient.birthDate"",
+                  ""method"": ""dateShift""
+                }
+              ],
+              ""parameters"": {
+                ""dateShiftKey"": ""123""
+              }
+            }";
+
+            Patient patient = new Patient();
+            patient.BirthDate = "2001-01-01";
+            IAnonymizer anonymizer = await CreateAnonymizerFromConfigContent(configurationContent);
+
+            ResourceElement resourceElement = anonymizer.Anonymize(new ResourceElement(patient.ToTypedElement()));
+            Patient anonymizedResource = resourceElement.Instance.ToPoco<Patient>();
+            Assert.Equal("2001-02-20", anonymizedResource.BirthDate);
+        }
+
+        [Fact]
+        public async Task GivenSubstituteAnonymizationConfig_WhenAnonymizeResource_ThenSubstitutedNodeShouldBeReturned()
+        {
+            string configurationContent =
+            @"
+            {
+              ""fhirPathRules"": [
+                {
+                  ""path"": ""Patient.name.family"",
+                  ""method"": ""substitute"",
+                  ""replaceWith"": ""test""
+                }
+              ]
+            }";
+
+            Patient patient = new Patient();
+            patient.Name.Add(HumanName.ForFamily("input"));
+            IAnonymizer anonymizer = await CreateAnonymizerFromConfigContent(configurationContent);
+
+            ResourceElement resourceElement = anonymizer.Anonymize(new ResourceElement(patient.ToTypedElement()));
+            Patient anonymizedResource = resourceElement.Instance.ToPoco<Patient>();
+            string resourceId = EncryptUtility.DecryptTextFromBase64WithAes(anonymizedResource.Id, Encoding.UTF8.GetBytes("704ab12c8e3e46d4bea600ef62a6bec7"));
+            Assert.Equal("test", anonymizedResource.Name.First().Family);
+        }
+
+        [Fact]
+        public async Task GivenPerturbAnonymizationConfig_WhenAnonymizeResource_ThenPerturbedNodeShouldBeReturned()
+        {
+            string configurationContent =
+            @"
+            {
+              ""fhirPathRules"": [
+                {
+                  ""path"": ""Condition.onset as Age"",
+                  ""method"": ""perturb"",
+                  ""span"": 0,
```

Comment:
nit: just curious whether it is normal to use `perturb` with `span: 0` since that means the value is not changed? Should we also have a test where `span` is set to a non-zero value and then we validate that the result is within the expected range?

Reply:
thanks for the comments, we want to have basic tests to cover in these tests to make sure integration with FHIR.